dhis-2/dhis-e2e-test/src/main/resources/config.properties

```diff
@@ -3,8 +3,8 @@
 instance.url=http://localhost:8080/api
 # properties for user set up on the instance.
 # user must be able to import metadata.
-user.default.username=system
-user.default.password=System123
+user.default.username=admin
+user.default.password=district
```

Comment:
Didn't I just recently change these to the values now replaced because otherwise e2e gives you trouble locally with demo database?